src/server/core/journey/route/graph.ts

```
@@ -7,16 +7,18 @@ import {
   MAX_DESTINATION_WALKING_DISTANCE,
 } from '@/lib/constants/config';
 import { TRANSFER_TIME } from '@/lib/constants/route-config';
-import { metroLines } from '@/lib/constants/metro-data';
+import { metroLines, walkingShortcuts } from '@/lib/constants/metro-data';
 import { graphCache } from '../../shared/cache';
+import {
+  calculateWalkingDuration,
+  calculateWalkingPenalty,
```

ESLint

[error] 'calculateWalkingPenalty' is defined but never used.

(@typescript-eslint/no-unused-vars)